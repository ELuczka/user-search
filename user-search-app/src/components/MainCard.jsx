import React from "react";

import { Box, Grid } from "@material-ui/core";
import UserSearch from "./UserSearch";

const MainCard = () => {
  return (
    <>
      <Box component={Grid} mt={6}>
        <Grid container spacing={2} direction="column" alignItems="center">
          <UserSearch />
        </Grid>
      </Box>
    </>
  );
};

export default MainCard;
