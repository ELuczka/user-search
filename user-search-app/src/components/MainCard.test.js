import React from 'react';

import renderer from 'react-test-renderer';

import MainCard from './MainCard';

it('Test', () => {
    const component = renderer
    .create(<MainCard/>)
    .toJSON();

    expect(component).toMatchSnapshot();
});