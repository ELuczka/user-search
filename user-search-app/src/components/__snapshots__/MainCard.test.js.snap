// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test 1`] = `
<div
  className="MuiGrid-root MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-5"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <div
          className="MuiPaper-root makeStyles-searchUser-2 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            onClick={[Function]}
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
            />
          </svg>
          <div
            className="MuiInputBase-root makeStyles-input-3"
            onClick={[Function]}
          >
            <input
              className="MuiInputBase-input"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              placeholder="Search for users"
              type="text"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-searchButton-4 MuiButton-containedPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Search
          </span>
        </button>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-5"
    />
  </div>
</div>
`;
